sonar-html-plugin/src/test/java/org/sonar/plugins/html/checks/accessibility/DomElementsShouldUseAutocompleteAttributeCorrectlyCheckTest.java

```diff
@@ -0,0 +1,118 @@
+
+    checkMessagesVerifier.verify(htmlSourceCode.getIssues())
+      .next().atLine(1).withMessage("DOM elements should use the \"autocomplete\" attribute correctly")
+      .next().atLine(2)
+      .next().atLine(3);
+
+    HtmlSourceCode jspSourceCode = TestHelper.scan(new File("src/test/resources/checks/DomElementsShouldUseAutocompleteAttributeCorrectlyCheck/bday/file.jsp"), check);
+
+    checkMessagesVerifier.verify(jspSourceCode.getIssues());
+
+    HtmlSourceCode phtmlSourceCode = TestHelper.scan(new File("src/test/resources/checks/DomElementsShouldUseAutocompleteAttributeCorrectlyCheck/bday/file.phtml"), check);
+
+    checkMessagesVerifier.verify(phtmlSourceCode.getIssues());
+
+    HtmlSourceCode vueSourceCode = TestHelper.scan(new File("src/test/resources/checks/DomElementsShouldUseAutocompleteAttributeCorrectlyCheck/bday/file.vue"), check);
+
+    checkMessagesVerifier.verify(vueSourceCode.getIssues())
+      .next().atLine(2).withMessage("DOM elements should use the \"autocomplete\" attribute correctly")
+      .next().atLine(3)
+      .next().atLine(4);
+  }
+
+  @Test
+  void on() {
+    HtmlSourceCode sourceCode = TestHelper.scan(new File("src/test/resources/checks/DomElementsShouldUseAutocompleteAttributeCorrectlyCheck/on.html"), new DomElementsShouldUseAutocompleteAttributeCorrectlyCheck());
+
+    checkMessagesVerifier.verify(sourceCode.getIssues())
+      .next().atLine(1).withMessage("DOM elements should use the \"autocomplete\" attribute correctly");
+  }
+
+  @Test
+  void tel() {
+    HtmlSourceCode sourceCode = TestHelper.scan(new File("src/test/resources/checks/DomElementsShouldUseAutocompleteAttributeCorrectlyCheck/tel.html"), new DomElementsShouldUseAutocompleteAttributeCorrectlyCheck());
+
+    checkMessagesVerifier.verify(sourceCode.getIssues())
+      .next().atLine(1).withMessage("DOM elements should use the \"autocomplete\" attribute correctly")
+      .next().atLine(2)
+      .next().atLine(3)
+      .next().atLine(4);
+  }
+
+  @Test
+  void withContext() {
+    HtmlSourceCode sourceCode = TestHelper.scan(new File("src/test/resources/checks/DomElementsShouldUseAutocompleteAttributeCorrectlyCheck/with-context.html"), new DomElementsShouldUseAutocompleteAttributeCorrectlyCheck());
+
+    checkMessagesVerifier.verify(sourceCode.getIssues())
+      .next().atLine(1).withMessage("DOM elements should use the \"autocomplete\" attribute correctly")
+      .next().atLine(2);
+  }
+
+  @Test
+  void withWebauthn() {
+    HtmlSourceCode sourceCode = TestHelper.scan(new File("src/test/resources/checks/DomElementsShouldUseAutocompleteAttributeCorrectlyCheck/with-webauthn.html"), new DomElementsShouldUseAutocompleteAttributeCorrectlyCheck());
+
+    checkMessagesVerifier.verify(sourceCode.getIssues())
+      .next().atLine(1).withMessage("DOM elements should use the \"autocomplete\" attribute correctly")
```

Comment:
```suggestion
      .next().atLine(1).withMessage("DOM elements should use the \"autocomplete\" attribute correctly.")
```

Reply:
No need to change, but there is no need to assert the issue message if it is static. Checking it once should be enough.

Reply:
Good point!

I will add the dot.